cli/src/error.rs

```diff
@@ -32,3 +32,9 @@ impl From<ClientError> for Error {
         Error::Client(Box::new(error))
     }
 }
+
+impl From<iota_sdk::client::stronghold::Error> for Error {
+    fn from(error: iota_sdk::client::stronghold::Error) -> Self {
+        Self::Client(Box::new(iota_sdk::client::Error::Stronghold(error)))
```

Comment:
Why box here but not on the spot above? Box is only needed for recursive errors and large enum things right?

Reply:
Not sure what you are asking, what is the "spot above" ?

Reply:
bindings/core/src/error.rs

Reply:
Ah, I see there is a `#[allow(clippy::large_enum_variant)]` there. We have an issue to change that already 👍🏻